✨ Embed Productboard

steffinchen opened the pull request:
## Summary

- Embed Productboard on /feedback
- Link to /feedback in menu drawer, close to the APPUiO Cloud status badge (as both feedback and status relate to the APPUiO Cloud)

Closes https://github.com/appuio/cloud-portal/issues/317

## Checklist

- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`,
      as they show up in the changelog
- [x] Update tests.
- [x] Link this PR to related issues.

<!--
Remove items that do not apply. For completed items, change [ ] to [x].

NOTE: these things are not required to open a PR and can be done afterwards,
while the PR is open.
-->

corvus-ch reviewed (approved):
The location of the Feedback menu entry might be easily overlooked. However, I do not have a suggestion for a better placing.

gerbermichi commented on src/app/productboard/productboard.component.scss:
```diff
@@ -0,0 +1,7 @@
+:host {
+  height: calc(100% + 2rem);
+  width: calc(100% + 4rem);
```
Does this work with the flexible menu? Can't you use flex-1?

steffinchen replied:
(resolving this as discussed offline)

tobru commented:
I'd remove the introduction text, doesn't feel it adds much value besides stealing space...

The page title somehow is different than on other pages:

![Screenshot_20220915_143812](https://user-images.githubusercontent.com/1695957/190406033-fb692aab-df27-4f71-ad47-4ec5c0d52166.png)
![Screenshot_20220915_143825](https://user-images.githubusercontent.com/1695957/190406039-64bc622e-2153-41ae-a327-42d9797744f2.png)

Note the bigger space above the title...

Regarding placement, I could live with this location, but we might try to place it more prominently right next to the help (?) menu?

tobru commented:
And I propose to set the URL to https://portal.productboard.com/vshn/1-vshn-product-portal/tabs/2-planned so that the "Planned" tab gets displayed first.

steffinchen commented:
@tobru I've changed the URL so it shows the "Planned" tab, and removed the intro text and extra spacing around the title. 
We don't really have the space in the top bar for another item, esp. on mobile we'd only have space for an icon (with no text) but there's no universally understood icon for feedback. 

Another option to drive users a bit more to the feedback & roadmap could be a pop-up at the bottom right, see below screenshot for a quick mockup. Ideally, we'd only show this to some users (e.g. when a frustration click-rage is detected, or for new-ish users, or experienced users, or controlled via an external feature toggle tool like [LaunchDarkly](https://launchdarkly.com/)...) but if you want to go this way I'd suggest we discuss the exact conditions in a separate ticket.

<img width="1056" alt="Screenshot 2022-09-20 at 09 54 09" src="https://user-images.githubusercontent.com/5945974/191205167-26e149e2-69da-470d-9f5a-2e55345e33eb.png">

corvus-ch commented:
@steffinchen Tobru is unavailable this week. I would say, this is good enough for now. We can pick up the placing, Icon and pop-up discussion any time we see a need for it.
